### PR TITLE
First draft of the engineering process.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
...
+
 This repository is used to track anything related to the platform that powers SUMO
+
+## Engineering Board
+
+Short description of the workflow and of the columns in the board.
+
+### Labels
+
+Each label in this board signifies a specific project.
```
Comment (LeoMcA) [inline README.md:11]: "project" being kitsune, m-response, etc. or "project" being responsive redesign, elastic, aaq, IA, etc.?

IMO milestones work better for the latter (you get a sense of progress, things can only belong to one) and labels work better for things that might cut across issues (sizing, bug, good-first-bug, etc.)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+
+### Columns
+
+#### Backlog column
+
+A bakclog of tasks that will be worked in the next three months.
```
Comment (LeoMcA) [inline README.md:17]: "bakclog" typo

File: README.md [hunk 3]
```diff
@@ -1,2 +1,54 @@
...
+Ideally meta-issues should be used here.
+
+#### Next items column
+
+Smaller prioritized backlog with cards that will be worked on in the two upcoming weeks. Should be sparsely populated and frequently updated.
+
```
Comment (LeoMcA) [inline README.md:23]: Do we need a triage/icebox column for things which aren't necessarily a priority, but come up while we're working, and might make a good contributor issue (for instance)?

File: README.md (hunk 1 above)
Comment (akatsoulas) [reply README.md:11]: Project being kitsune, respond something new along the way. Milestones need also to go in this and of course describe the process. I wanted to start with a skeleton README that we can build upon

**Pull request merged**